topi/python/topi/x86/conv2d.py

```diff
@@ -28,10 +34,68 @@ def traverse(op):
                 data_pad = data
                 data = data_pad.op.input_tensors[0]
 
+            n_pad, c_pad, h_pad, w_pad = data_pad.op.axis
+            pad_fused = s[data_pad].fuse(n_pad, c_pad)
+            s[data_pad].parallel(pad_fused)
             C = conv
             n, c, h, w = C.op.axis
-            s[C].parallel(c)
-            s[C].pragma(n, "parallel_launch_point")
+            rc, ry, rx = C.op.reduce_axis
+            fused = s[C].fuse(n, c)
+            s[C].parallel(fused)
+            wo, wi = s[C].split(w, factor=16)
+            s[C].reorder(fused, rc, h, wo, ry, rx, wi) # move rc to outer loop
+            s[C].unroll(rx)
+            s[C].unroll(ry)
+            s[C].vectorize(wi)
```

Comment:
I tested this schedule on AWS C5 instance (skylake-avx512 4-cpu) which does not perform well, about `13 ms` for conv only.
My input is `(1, 64, 56, 56)` and kernel size `(64, 64, 3, 3)`

Reply:
Looks like the final s[C].vectorize(wi) is not working for this input size (56).
For my use cases, all of input width and height is power of two. I wrote my schedules assuming such input.